File: crates/deslop/tests/fixtures/python-extract-type1/metrics.py
def total_with_tax(amounts, tax_rate):
    total = 0
    for amount in amounts:
        taxed = amount * tax_rate // 100
        total = total + amount + taxed
    if total > 10000:
        total = 10000
    return total


def subtotal_with_tax(amounts, tax_rate):
    total = 0
    for amount in amounts:
        taxed = amount * tax_rate // 100
        total = total + amount + taxed
    if total > 10000:
        total = 10000
    return total

File: crates/deslop/tests/fixtures/rust-extract-type1/metrics.rs
pub fn total_with_tax(amounts: &[usize], tax_rate: usize) -> usize {
    let mut total = 0;
    for amount in amounts {
        let taxed = amount * tax_rate / 100;
        total += amount + taxed;
    }
    if total > 10_000 {
        total = 10_000;
    }
    total
}

pub fn subtotal_with_tax(amounts: &[usize], tax_rate: usize) -> usize {
    let mut total = 0;
    for amount in amounts {
        let taxed = amount * tax_rate / 100;
        total += amount + taxed;
    }
    if total > 10_000 {
        total = 10_000;
    }
    total
}
